src/library/parserm3u.cpp

```diff
@@ -110,13 +110,10 @@ QString ParserM3u::getFilepath(QTextStream* stream, QString basepath)
 
         if (!textline.contains("#")) {
             filename = textline;
+            filename.replace('\\', '/');
```

Comment:
All those replacements should be obsolete when using [QUrl::fromLocalFile()](http://doc.qt.io/qt-5/qurl.html#fromLocalFile)

Reply:
According to 
http://www.scvi.net/pls.htm
It can be a plain path or an URL 
Does  QUrl::fromLocalFile() accept URLs ? 

We need actually the local file the URL round trip is only done to normalize URL encoded strings. This works like a charm with QT4 so the fix aims to just go back to the working state.

Reply:
How about using QUrl::fromUserInput() with options = QUrl::AssumeLocalFile? Then we might still need to convert the backslashes, though. We definitely need tests for all the possible variants instead of discussing them here ;)